npm/angular/src/app/material-button/material-button.component.ts

```diff
@@ -1,5 +1,4 @@
 import { Component, OnInit } from '@angular/core'
-import '@angular/material/prebuilt-themes/indigo-pink.css'
```

Comment:
There is no reason for this import to be made here (it's currently doing nothing) and it is now throwing a compile time error because of the switch to IVY